internal/config/config.go

```diff
@@ -38,6 +38,15 @@ func MustEnableUnitTestingMode() {
 }
 
 func EnableUnitTestingMode() error {
+	return EnableUnitTestingModeFull(true)
+}
+
+// - Sets configuration options
+// - Changes base directory to a temp directory
+// - Sets logging to debug if not already set
+// - Copies test data into the working directory.
+// This unit testing mode supports loading environment variables through the loadEnv argument.
```

Comment:
I don't like this sentence, it does not seem really helpful and I think it is trying to be too fancy.
Just be direct: "If loadEnv is true ...".